File: app/src/main/java/com/hubwallet/apptspos/Utils/Models/ServiceModel/ServiceData.java
package com.hubwallet.apptspos.Utils.Models.ServiceModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ServiceData {

    @SerializedName("service_data")
    @Expose
    private List<ServiceDatum> serviceData = null;

    public List<ServiceDatum> getServiceData() {
        return serviceData;
    }

    public void setServiceData(List<ServiceDatum> serviceData) {
        this.serviceData = serviceData;
    }

}

File: app/src/main/java/com/hubwallet/apptspos/retrofit/login/ServiceData.kt
package com.hubwallet.apptspos.retrofit.login


import com.google.gson.annotations.SerializedName
import androidx.annotation.Keep

@Keep
data class ServiceData(
    @SerializedName("service_id")
    var serviceId: String = "",
    @SerializedName("service_name")
    var serviceName: String = ""
)